Godeps/_workspace/src/github.com/openshift/openshift-sdn/pkg/ovssubnet/controller/multitenant/bin/openshift-sdn-multitenant-setup.sh

```diff
@@ -114,6 +120,7 @@ function setup() {
     ip link set vovsbr up
     ip link set vlinuxbr txqueuelen 0
     ip link set vovsbr txqueuelen 0
+    brctl addif lbr0 vlinuxbr
```

Comment:
Agreed. Though in a OVS in a container case this has happened before. Not an issue we need to address right now maybe. 
As a quick fix otherwise, we just need to check if vlinuxbr is valid or not and return that as a check for setup_required.